### PR TITLE
feat: Add the ability to change the agent availability status

### PR DESCRIPTION
**Pull request by muhsin-k**
Fixes https://linear.app/chatwoot/issue/CW-382/force-logout-agents

https://user-images.githubusercontent.com/12408980/230562135-c2072964-5c79-41ad-bc1f-647ac19ba9f9.mov

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/agents/EditAgent.vue [hunk 1]
```diff
@@ -65,6 +83,8 @@ import WootSubmitButton from '../../../../components/buttons/FormSubmitButton';
 import Modal from '../../../../components/Modal';
 import Auth from '../../../../api/auth';
 
+const AVAILABILITY_STATUS_KEYS = ['online', 'busy', 'offline'];
```
Comment (scmmishra) [inline app/javascript/dashboard/routes/dashboard/settings/agents/EditAgent.vue:86]: can we move this to `dashboard/constants.js`?

Comment (muhsin-k) [reply app/javascript/dashboard/routes/dashboard/settings/agents/EditAgent.vue:86]: Sure.

File: app/javascript/dashboard/i18n/locale/en/agentMgmt.json [hunk 2]
```diff
@@ -74,6 +75,11 @@
           "LABEL": "Email Address",
           "PLACEHOLDER": "Please enter an email address of the agent"
         },
+        "AGENT_AVAILABILITY": {
+          "LABEL": "Availability",
+          "PLACEHOLDER": "Please select an availability",
```
Comment (iamsivin) [inline app/javascript/dashboard/i18n/locale/en/agentMgmt.json:80]:
```suggestion
          "PLACEHOLDER": "Please select an availability status",
```
Thoughts?

**Review by iamsivin — APPROVED** [review]
LGTM. Added a small comment
https://www.loom.com/share/55951db313104c5f92dc821b701aace3

**Review by scmmishra — APPROVED** [review]

**Pull request merged**